src/content/posts/cfp-workshop-recap.md

```diff
@@ -0,0 +1,84 @@
+
+## Successful submission patterns
+
+There are common successful submission patterns seen at conferences, adjusting a submission title to fit into a common pattern may increase chances of a submission being voted in as a talk at DDD Sydney.
+
+### Asking a question
+
+> What do microservices really look like in prod?
+
+> How would you get started with Docker?
+
+This technique invites the audience to be curious about a specific subject. When asking a question, the speaker has already outlined the outcome of the talk and the audience is seeking an answer to the question in the title or at least enough information to answer the question themselves.
+
+### A respectful challenge to the audience
+
+> Stop mocking, start testing
+
+> Zero to hero with Docker in 1 hour
+
+Talks like this challenge the audience on a certain behaviour or practice. When reading a title like this, the audience desires to know why a certain behaviour or belief may not be ideal, this leads to a desire on how to improve moving into the future.
+
+### A play on words
+
+> Docker FROM scratch
+
+Playing on words or syntax attracts the audience as there is often an element of clever humour associated with the talk title. Please note: titles which play on words have to be inclusive enough to be understood by enough people to be accepted as a conference talk.
+
+### From the trenches
+
+> DDD from the trenches
+
+> Docker from the trenches
+
+From the trenches talks appear at conferences because they outline both positive and negative experiences of a particular practice or technology. Attendees appreciate hearing about difficulties associated with particular practices or technologies as they are honest, come from real life experience, and can be used to minimise future difficulties if attendees wish to adopt them.
+
+### Personal talks
+
+> Overcoming fear, anxiety and doubt as a developer
+
+> How I became brave enough to use Docker
+
+At every conference, there's one or two talks that are personal and come from the heart. Attendees like these talks as they provide a break from technically heavy talks, at the same time, the delivery of talks like this can be hard as the speaker is making themselves vulnerable and revealing a lot of emotion to the audience.
+
+## Conclusion
+
+There are a variety of techniques that are used to make language communicate a message more effectively. When writing titles and abstracts for a submission, it's important to take these into consideration in addition to examining common succesful submission patterns. Be sure to checkout past DDD Sydney agendas to see these common patterns and what works for you!
```

Comment:
Just a suggestion to make the first sentence of the conclusion concise: "There are a variety of language techniques used to communicate a message effectively."

Reply:
Will fix

Reply:
i made it There are a variety of language techniques used to communicate a message more effectively